src/vm/jvm/QAST/Compiler.nqp

```diff
@@ -2396,7 +2396,7 @@ QAST::OperationsJAST.map_classlib_core_op('normalizecodes', $TYPE_OPS, 'normaliz
 QAST::OperationsJAST.map_classlib_core_op('codes', $TYPE_OPS, 'codes', [$RT_STR], $RT_INT);
 
 QAST::OperationsJAST.map_classlib_core_op('codepointfromname', $TYPE_OPS, 'codepointfromname', [$RT_STR], $RT_INT);
-QAST::OperationsJAST.map_classlib_core_op('getstrfromname', $TYPE_OPS, 'getstrfromname', [$RT_STR], $RT_STR);
+QAST::OperationsJAST.map_classlib_core_op('strfromname', $TYPE_OPS, 'getstrfromname', [$RT_STR], $RT_STR);
```

Comment:
You've changed method in Ops.java from `getstrfromname` to `strfromname`, so the next `'getstrfromname'` should be changed, too. I think.

Reply:
Good catch. I've force-pushed a correction.